package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "wikit",
+  "private": false,
```

Comment:
Shouldn't this one say `true`? We publish sub packages, if ever, right?

Reply:
Ha. I have no idea how I got this one wrong.

Reply:
Thank you for paying attention!